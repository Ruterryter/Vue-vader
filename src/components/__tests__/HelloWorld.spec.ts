import { shallowMount } from '@vue/test-utils'
import HelloWorld from '../HelloWorld.vue'
import { expect, describe, it } from '@jest/globals'

describe('HelloWorld.vue', () => {
  it('matches the snapshot', () => {
    const msg = 'Hello, Jest and Vue!'
    const wrapper = shallowMount(HelloWorld, {
      props: { msg }
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
