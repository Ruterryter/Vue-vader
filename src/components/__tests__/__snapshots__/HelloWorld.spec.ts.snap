// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue matches the snapshot 1`] = `
"<div class="greetings">
  <h1 class="green">Hello, Jest and Vue!</h1>
  <h3> Youâ€™ve successfully created a project with <a href="https://vitejs.dev/" target="_blank" rel="noopener">Vite</a> + <a href="https://vuejs.org/" target="_blank" rel="noopener">Vue 3</a>. What's next? </h3>
</div>"
`;
